.github/workflows/test-unit-pr.yml

```
@@ -0,0 +1,106 @@
+  cancel-in-progress: true
+
+jobs:
+  parse-components:
+    name: Parse Affected Components
+    runs-on: ubuntu-latest
+    outputs:
+      testComponents: ${{ steps.parseTitle.outputs.testComponents }}
+    steps:
+      - name: Parse Title
+        id: parseTitle
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)